plugins/index.js

```diff
@@ -203,4 +204,6 @@ export async function loadPluginsIfNecessary() {
             removePlugin(oldManifest);
         }
     });
+
+    fetchMobilePluginIntegrations()(store.dispatch, store.getState);
```

Comment:
Same comment here about dispatching actions
```suggestion
    store.dispatch(fetchMobilePluginIntegrations());
```